src/client/utils/tracking/gtm.ts

```diff
@@ -38,6 +39,7 @@ export const trackOfferGTM = (
         number_of_people: offerData.person.householdSize,
         insurance_price: parseFloat(offerData.cost.monthlyNet.amount),
         currency: offerData.cost.monthlyNet.currency,
+        member_id: offerData.memberId,
```

Comment:
Could we maybe do something like this, only passing memberId if it's defined? :)
```suggestion
        ...(offerData.memberId && { member_id: offerData.memberId }),
```